descholar-ceo/AnnounceIT
# #170726755 Users all announcements

#### What does this PR ​ do​ ?
User should be able to view all his announcements

#### Description of Task to be completed?
* User sees list of all his announcements on his/her profile page

 #### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

*Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/user-profile.html`, will be redirected to the signin page and click on signin button, you will be immediatelly to the user profile-page

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `user-profile.html` and right click and open with your preferred browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170726755](https://www.pivotaltracker.com/n/projects/2429498)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-18 20-15-18](https://user-images.githubusercontent.com/56886064/72668422-514b8880-3a2f-11ea-96d1-14e2d556ebe5.png)

#### Questions:
No Question
